EpsilonDashboard/EpsilonDashboard.cpp

```diff
@@ -12,10 +14,33 @@ EpsilonDashboard::EpsilonDashboard(int& argc, char** argv)
     , communicationContainer_(new CommunicationContainer())
     , businessContainer_(new BusinessContainer(*communicationContainer_, *dataContainer_))
     , presenterContainer_(new PresenterContainer(*dataContainer_))
-    , viewContainer_(new ViewContainer(*presenterContainer_))
 {
+    char mode = parseCommandLineArgs();
+    viewContainer_.reset(new ViewContainer(*presenterContainer_, mode));
```

Comment:
Ah I see your point with the reset, I forgot that was how QScopedPointer worked.